nncf/common/sparsity/controller.py

```diff
@@ -0,0 +1,34 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from abc import ABC, abstractmethod
+
+
+class SparsityController(ABC):
```

Comment:
I absolutely agree with you. I fixed it. I also removed the `@abstractmethod` decorator because it should be added in 51104 (i.e. when `CompressionAlgorithmController` will be inherited from `abc.ABC`).